config/metadata/configMeta.yaml

```diff
@@ -399,6 +399,27 @@ groups:
           Decreasing this will check the trace cache for timeouts more
           frequently.
 
+      - name: MaxExpiredTraces
+        type: int
+        valuetype: nondefault
+        default: 5000
+        reload: true
+        validations:
```

Comment:
Does this need a `firstversion` entry for `2.8.5`?

Reply:
Since we are now including this into 2.9, I updated the `firstVersion` to be `2.9`